spec/system/admin/admin_dashboard_presented_spec.rb

```diff
@@ -0,0 +1,23 @@
+require "rails_helper"
+
+RSpec.describe "Admin dashboard is presented", type: :system do
+  let(:admin) { create(:user, :super_admin) }
+  let(:user) { create(:user) }
+
+  before { Bullet.raise = false }
+  after { Bullet.raise = true }
```

Comment:
ahah I think I know why you did this, eventually I'll figure out why Bullet is complaining, I have an idea, but I haven't had the time to circle back to that

Reply:
Thanks @rhymes! 

Yep, from what I found it feels like the `Administrate` gem handles de `index` action (bc not explicitly found in the controllers) and it eager loads stuff, in this case it was Article's comments. Of course that's just the first step into the rabbit hole, hope it helps in any way when you do circle back to it though  😅